src/app/components/CollapsibleNavigation/index.tsx

```diff
@@ -0,0 +1,139 @@
+};
+
+const CollapsibleNavigation = ({
+  navigationSections,
+}: CollapsibleNavigationProps) => {
+  const [openSection, setOpenSection] = useState<string | null>(null);
+  const isHydrated = useHydrationDetection();
+  const activeNavItemRef = useRef<HTMLAnchorElement | null>(null);
+
+  const handleNavClick = (
+    e: React.MouseEvent<HTMLAnchorElement>,
+    section: CollapsibleNavigationSection,
+  ) => {
+    if (section.href) {
+      return;
+    }
+
+    e.preventDefault();
+    const isActive = openSection === section.id;
+
+    if (isActive) {
+      setOpenSection(null);
+      activeNavItemRef.current = null;
+    } else {
+      activeNavItemRef.current = e.currentTarget;
+      setOpenSection(section.id);
+    }
+  };
+
+  const handleClose = (e: React.MouseEvent) => {
+    e.preventDefault();
+    setOpenSection(null);
+
+    if (activeNavItemRef.current) {
+      activeNavItemRef.current.focus();
+      activeNavItemRef.current = null;
+    }
+  };
+
+  return (
+    <nav role="navigation">
+      <ul role="list" css={styles.navList}>
+        {navigationSections.map(section => {
+          const isActive = Boolean(openSection === section.id);
+          const shouldShowSubNav = isHydrated ? isActive : true;
+          const isLink = section.href;
+
+          const navigationLinkId = `nav-${section.id}`;
+          const subNavigationTitleId = `subnav-title-${section.id}`;
+          const subNavigationId = section.id;
+
+          return (
+            <React.Fragment key={section.id}>
+              <li css={styles.navItem} role="listitem">
+                <a
+                  id={navigationLinkId}
+                  href={section.href || `#${section.id}`}
+                  onClick={e => handleNavClick(e, section)}
+                  className="focusIndicatorRemove"
+                  css={[styles.navLink, isActive && styles.navLinkActive]}
+                  role={isLink ? 'link' : 'button'}
+                  {...(isActive && { 'aria-current': 'true' })}
+                  {...(!isLink && { 'aria-expanded': isActive })}
+                  {...(!isLink && { 'aria-controls': subNavigationId })}
+                >
+                  {section.title}
+                </a>
+              </li>
+
+              {section.links && shouldShowSubNav && (
+                <li
+                  id={subNavigationId}
+                  css={[styles.subNav, !isHydrated && styles.subNavNoJs]}
+                  role="region"
+                  aria-labelledby={subNavigationTitleId}
+                >
+                  <div css={styles.subNavHeader}>
+                    <Heading
+                      level={3}
+                      id={subNavigationTitleId}
+                      css={styles.subNavTitle}
+                    >
+                      {section.title}
+                    </Heading>
+                    <a
+                      aria-label={`Close ${section.title} submenu`}
```

Comment:
Should we use translations here (maybe not in this ticket, but if we were to use this component instead the current nav, then we would need it to be translatable - can you please add a follow up ticket?). 

This also means that we would use simpler language - does the audience know what a 'submenu' is? In fact, should we remove this word completely?


```suggestion
                      aria-label={`Close ${section.title}`}
```

Reply:
UX asked to include the `submenu` to improve clarity for users.

I've added this to the Global Navigation spike - [WS-1141](https://bbc.atlassian.net/browse/WS-1141). We can revisit this when planning to use it with more services